File: HalfWerk.CommonModels/DsBestelService/Models/BestelRegel.cs
namespace HalfWerk.CommonModels.DsBestelService.Models
{
    public class BestelRegel
    {
        public long Id { get; set; }
        public long Artikelnummer { get; set; }
        public string LeverancierCode { get; set; }
        public string Naam { get; set; }
        public int Aantal { get; set; }
        public decimal PrijsExclBtw { get; set; }
        public decimal PrijsInclBtw { get; set; }
        public decimal RegelTotaalExclBtw { get; set; }
        public decimal RegelTotaalInclBtw { get; set; }
    }
}

File: HalfWerk.FeWebshop/src/app/models/bestelregel.ts
export class BestelRegel {
  id: number;
  afbeeldingUrl: string;
  artikelnummer: number;
  leverancierCode?: string;
  naam: string;
  aantal: number;
  prijs?: number;
  prijsExclBtw?: number;
  prijsInclBtw?: number;
  regelTotaalExclBtw?: number;
  regelTotaalInclBtw?: number;
}
